rfc/rfc012-nuts-company-credential.md

```diff
@@ -0,0 +1,119 @@
+# RFC012 Nuts Company Credential
```

Comment:
I have the feeling the issuer/use-case and the credential is tightly coupled. I can imagine several issuers with different trust levels for the same credential:
The DID subject
A vendor about its customer
The KVK
The Nuts foundation

All contain the same credential fields. Why prefix a CompanyCredential with Nuts and tie it to this specific use-case: bootstrapping an identity? Perhaps a DID can have multiple CompanyCredentials, one issued by itself and one issued by the chamber of commerce.

Reply:
All those would have a different type.

Reply:
Imagine an AlumniCredential which several universities are issueing. I don't think it is ment to become an MITAlumniCredential, StanfortAlumniCredential etc. I think each of these universities are going to issue the same AlumniCredential. So why should we take another approach and couple the issuer to the type? I would say the trustlevel of the issuer depends on the issuer. Not on the type.

Reply:
Or let me rephrase that. If the contents if the `credentialSubject` is different then the type must also differ. This is a basic consequence of the use of @context.